RELEASES.md

```diff
@@ -0,0 +1,87 @@
+# Releases
+
+## Versioning
+
+SOCI versions follows the `Major.Minor.Patch-PreRelease` scheme of [Semantic Versioning](https://semver.org).
+
+### `soci CLI / soci-snapshotter / SOCI library`
+
+The soci CLI, soci-snapshotter, and library all share the same version to make it easy to understand compatibility.
+
+### `ztoc data format`
```

Comment:
ztoc -> zTOC (assuming zTOC is our preferred format).

Also the code block `` is not needed as `zTOC data format` is not   code

Reply:
fixed